### PR TITLE
Added testing to MC Questions

### PR DESCRIPTION
**Pull request by OmarAgamy9**
Added few tests for the MCQ's.
Changed the naming of the MCQ class to MCQModel

### CODE REVIEW

File: src/main/java/com/SurveyMonkey/model/questions/Questions.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import java.util.HashMap;
 
 @Entity
-public class Questions {
+abstract public class Questions {
```
Comment (keving3ng) [inline src/main/java/com/SurveyMonkey/model/questions/Questions.java:9]: There is already an abstract type called QType, this class was supposed to be what CreateSurvey is right now (a class to hold a list of questions).

**Review by keving3ng — CHANGES_REQUESTED** [review]
the jars under lib/ that are in this branch should not be commited. please omit them in your gitignore and untrack them in git to remove

see this https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign

File: src/test/java/com/SurveyMonkey/TakerControllerTests.java [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-	@Autowired
-	private TakerController takerPageController;
-
-	@Test
-	void contextLoads() throws Exception{
-		assertThat(takerPageController).isNotNull();
-	}
-
-}
+//package com.SurveyMonkey;
```
[the remark this answers is not in the document]
Comment (hassanidea) [reply src/test/java/com/SurveyMonkey/TakerControllerTests.java:1]: I will accept this pull request and I will make the changes necessary

**Review by hassanidea — APPROVED** [review]
Abstract class will be modified

**Review by keving3ng — APPROVED** [review]

**Pull request merged**